purview/models/koji.py

```diff
@@ -0,0 +1,55 @@
+from neomodel import (
+    StructuredNode, StringProperty, IntegerProperty, UniqueIdProperty,
+    BooleanProperty, DateTimeProperty, FloatProperty, RelationshipTo)
+
+
+class KojiBuild(StructuredNode):
+    id_ = UniqueIdProperty(db_property='id')
+    tasks = RelationshipTo('KojiTask', 'TRIGGERS')
+    creation_event_id = IntegerProperty(required=True)
+    volume_id = IntegerProperty()
+    owner_id = IntegerProperty()
+    package_id = IntegerProperty(required=True)
+    task_id = IntegerProperty(required=True)
+    build_id = IntegerProperty()
```

Comment:
Not sure, but the `build_id` should probably be the `UniqueIdProperty` of the label/node. What does `id_` currently stand for if it's not the `build_id`?

Reply:
In our call, @mprahl suggested removing this and storing the "brew build id" from the brew system in our `id_` UniqueIdProperty.

We should test that a KojiBuild with id_ 42 doesn't conflict with a FreshmakerEvent with id_ 42.